webapp/src/components/expanded_view/component.tsx

```diff
@@ -1355,10 +1388,18 @@ const CloseButton = styled.button`
     }
 `;
 
-const Overlay = styled.div`
+const ReactionOverlay = styled.div`
     position: absolute;
     bottom: 96px;
     display: flex;
     flex-direction: column;
     gap: 12px;
 `;
+
+const LiveCaptionsOverlay = styled.div`
+    position: absolute;
+    width: 100vw;
```

Comment:
Does this mean it will go on top of other elements like participants list, call thread?

Reply:
It goes on top of reactions and the participant list, but not over the thread view. I can _probably_ figure out how to put it on top of the thread view if needed, but I thought we might not want to do that, because having captions block the thread might be undesirable. I'm fine either way, so maybe @matthewbirtch can let us know which is best?

Reply:
@cpoile @streamer45 the caption should definitely not overlap on top of the thread RHS. The captions should stay within the bounds of the main call room area. 
![image](https://github.com/mattermost/mattermost-plugin-calls/assets/2040554/7ce6de77-eb9a-4f95-a299-19859306aad1)

Reply:
@matthewbirtch Thank you. How about the case when the participant list is active?

Reply:
After experimenting with both, I think it looks better if the captions are centred in the expanded view main area (with the blue background). That is: (we can adjust the max width etc.)
<img width="2035" alt="image" src="https://github.com/mattermost/mattermost-plugin-calls/assets/1490756/e5e86dc3-f36e-4ede-9d35-07269c2a7a6b">

Reply:
Yep, what you've done here makes the most sense to me - centered within the main area. Although, would we have this much text on the screen at one time? I assume we would cap it at 2 lines of text at a time.

For a max-width, I would play with something like 600px?

Reply:
@matthewbirtch As for the amount of text -- these are three people talking at once, so I don't think it makes sense to artificially limit the number of captions, do you? This is probably not going to happen often. Generally there is a 5 second wait until the last caption gets removed, so unless there's overlapping conversations, we usually will only see one or two captions at the same time.

600 px sounds good -- I'll play around it some more during testing and post some screenshots.

Reply:
> these are three people talking at once, so I don't think it makes sense to artificially limit the number of captions, do you

Oh, I see. I didn't make that connection at first - I should have looked closer. How many people are we tracking at one time? If we go much beyond 3, the screen would just fill with captions :) As you mentioned, not a likely scenario, but we may want to come up with some kind of limit to prevent that. Probably not a high priority fix right now though

Reply:
There's no limit to the number of captions we display at one time, but naturally there won't be too many. They disappear after 5 seconds of inactivity. I think this is the kind of thing we should experiment with with real-world usage and then we'll have a better sense of sense of what limits make sense. With the test scripts (with up to 3 people talking simultaneously, or with more natural one-after-another conversations) the number of captions is never overwhelming.